tests/topotests/nhrp_topo/r1/nhrp4_cache.json

```diff
@@ -9,7 +9,6 @@
       "protocol":"10.255.255.2",
       "nbma":"10.2.1.2",
       "claimed_nbma":"10.2.1.2",
-      "used":false,
```

Comment:
We have this value unconditionally, any reason to remove it from here?

Reply:
The 'used' attribute is dynamic and relies on the state of the
    system neighbor entry attached to the protocol address. Periodically,
    the value is set to false, when receiving a NUD_PROBE notification
    from netlink. Then, NHRP re-programs the link entry and re-sets the
    'used ' value to true.
    
    Because that value may be intermittently set to false, it is better to
    remove it from the tests.